docs/server/source/production-deployment-template/add-node-on-kubernetes.rst

```diff
@@ -144,15 +262,20 @@ Create the required Deployment using:
 You can check its status using the command ``kubectl get deploy -w``
 
 
-Step 6: Restart the Existing BigchainDB Instance(s)
+Step 8: Restart the Existing BigchainDB Instance(s)
 ---------------------------------------------------
 
-Add the public key of the new BigchainDB instance to the keyring of all the
-existing BigchainDB instances and update the BigchainDB instances using:
+Add the public key of the new BigchainDB instance to the ConfigMap ``bdb-keyring``
+variable of existing BigchainDB instances, update the ConfigMap of the existing
+BigchainDB instances and update the instances respectively:
 
 .. code:: bash
 
-   $ kubectl --context ctx-1 replace -f bigchaindb-dep.yaml 
+   $ kubectl --context ctx-1 apply -f configuration/config-map.yaml
+   $ kubectl --context ctx-1 replace -f bigchaindb/bigchaindb-dep.yaml
```

Comment:
Should we use a `--force` here?
Ref: https://kubernetes.io/docs/concepts/cluster-administration/manage-deployment/ under `Disruptive Updates`.